.github/workflows/crucible-merged.yaml

```diff
@@ -0,0 +1,25 @@
+name: crucible-merged
+
+on:
+  pull_request_target:
+    types: [ closed ]
+    branches: [ master ]
+    paths-ignore:
+      - LICENSE
+      - '**.md'
+      - .github/workflows/run-crucible-tracking.yaml
+      - .github/workflows/faux-crucible-ci.yaml
+      - 'docs/**'
+  workflow_dispatch:
+
+jobs:
+  call-tool-crucible-ci_if-merged:
+    if: github.event.pull_request.merged == true
```

Comment:
...and then this ensures it only continues if the PR was actually merged.

Reply:
This is the workflow that runs when a PR is merged and uses the "production" repos instead of the CI repos.  Properly running for a merged PR is a two part requirement (`PR closed && PR merged`) since a PR can be closed without merging.

Reply:
I get it now thanks